doc/reference/devices_unix_block.md

```diff
@@ -1,20 +1,25 @@
 (devices-unix-block)=
 # Type: `unix-block`
 
-Supported instance types: container
+```{note}
+The `unix-block` device type is supported for containers.
+It supports hotplugging.
+```
 
-Unix block device entries simply make the requested block device
-appear in the instance's `/dev` and allow read/write operations to it.
+Unix block devices make the specified block device appear  as a device in the instance (under `/dev`).
+You can read from the device and write to it.
 
-The following properties exist:
+## Device options
 
-Key         | Type      | Default           | Required  | Description
-:--         | :--       | :--               | :--       | :--
-`gid`       | int       | `0`               | no        | GID of the device owner in the instance
-`major`     | int       | device on host    | no        | Device major number
-`minor`     | int       | device on host    | no        | Device minor number
-`mode`      | int       | `0660`            | no        | Mode of the device in the instance
-`path`      | string    | -                 | no        | Path inside the instance (one of `source` and `path` must be set)
-`required`  | bool      | `true`            | no        | Whether or not this device is required to start the instance
-`source`    | string    | -                 | no        | Path on the host
-`uid`       | int       | `0`               | no        | UID of the device owner in the instance
+`unix-block` devices have the following device options:
+
+Key         | Type      | Default           | Description
+:--         | :--       | :--               | :--
+`gid`       | int       | `0`               | GID of the device owner in the instance
+`major`     | int       | device on host    | Device major number
+`minor`     | int       | device on host    | Device minor number
+`mode`      | int       | `0660`            | Mode of the device in the instance
+`path`      | string    | -                 | Path inside the instance (one of `source` and `path` must be set)
+`required`  | bool      | `true`            | Whether this device is required to start the instance
```

Comment:
It would be nice to add a note about `required=false` that if the device later appears on the host it will be automatically passed into the container even if its already started (hotplugging).

Reply:
You can test this yourself by plugging in a usb thumbdrive and then sharing the device (in /dev) from the host into the container with `required=false` and then try unplugging/plugging it back in.

Reply:
Is this only for `unix-block`, or also for `unix-char`?

Reply:
> Is this only for `unix-block`, or also for `unix-char`?

Oops, ignore, just saw the other comment. ;)